### DISCUSSION TITLE
Why is this package so much bigger than dayjs?

### DISCUSSION THREAD

**Discussion by riwu** [js-fns/date-fns#2138] [Ideas]
https://github.com/you-dont-need/You-Dont-Need-Momentjs#brief-comparison
`dayjs` (OO and no tree-shaking) is only 2.6kb while this package is easily 10kb+ even with tree-shaking (built and tested myself).
Is there anything that can be done to reduce this package to the size of `dayjs`?

**Comment by farfromrefug** [comment]
Could someone in the date-fns team answer that. My webpack report says 141kb for date-fns against 10kb for day-js !!

**Comment by Feshchenko** [comment]
What I see from the high perspective.
dayjs is a monolithic solution with limited functionality. But when you do
```
import dayjs from 'dayjs';

const NOW = dayjs();
```
It will bring all 6kb of its code to your bundle.
But when you import some function from date-fns - it will adds only this code and everything else will be tree-shaked.
In this case, you need to analyse the final bundle size :)
